apps/tsconfig.build.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I'd love a comment in this file tying this back to `tsconfig.json` and indicating that its also used in webpack.config.js for the ts-loader.

Reply:
I had a comment here but removed it because github highlighted it in red (because technically comments aren't allowed in json). It didn't actually seem to cause any issues in the editor or in using the file, so I can add it back if we are ok with them being flagged as invalid in some places.